libraries/angularjs/src/advanced-tests.js

```diff
@@ -17,15 +17,25 @@ describe("advanced support", () => {
   );
 
   describe("attributes and properties", () => {
+    const prep = el => {
+      return compile(el)(scope)[0];
+    }
+
     it("will pass array data as a property", function() {
       this.weight = 2;
-      let data = wc.arr;
-      expect(data).toEqual(["A", "n", "g", "u", "l", "a", "r"]);
+      let root = prep("<comp-with-props>")
+      scope.$digest()
+      let wc = root.querySelector('#wc')
+      let data = wc.arr || wc.getAttribute('arr')
```

Comment:
It shouldn't be calling `getAttribute` here. For arrays and objects, the test is trying to assert that they are always passed as JS properties.

Reply:
👍